ui/app/src/components/PreviewAudiencesPanel/AudiencesPanelUI.tsx

```diff
@@ -67,14 +68,26 @@ const messages = defineMessages({
   }
 });
 
-const getDefaultModel = (fields: LookupTable<ContentTypeField>) => {
+const getDateTimeString = (date: Date): string => {
+  const pieces = get8601Pieces(date);
+  return create8601String(pieces[0], pieces[1], pieces[2]);
+};
+
+const getDefaultModel = (fields: AudiencesPanelUIProps['fields']) => {
   const props = {};
 
   Object.keys(fields).forEach((fieldId: string) => {
     const propValue = fields[fieldId].defaultValue;
     props[fieldId] = propValue;
-  });
 
+    if (fields[fieldId].type === 'date-time') {
+      // If there is no default value set for date-time, set it to the current date/time
+      if (!propValue) {
+        props[fieldId] = getDateTimeString(new Date());
+      }
+      props[`${fieldId}_tz`] = fields[fieldId].timeZone;
+    }
+  });
```

Comment:
This would make this field essentially non nullable, right? Need to check with Russ if that's a problem. It sounds like it could.